### DISCUSSION TITLE
Benefits application page is not working

### DISCUSSION THREAD

**Discussion by ber-serker** [community/community#44607] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

I tried to sign up for the benefits in GitHub education but after completing the first page (https://education.github.com/discount_requests/pack_application), it froze after submitting the form. Tried it several times. Different browsers, different networks, clearing cookies.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ber-serker, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it for you. Good luck!
